docs/labelbox/embedding.rst

```diff
@@ -0,0 +1,7 @@
+Embedding
+===============================================================================================
+
+.. automodule:: labelbox.schema.embedding
+   :members:
+   :exclude-members: EmbeddingVector
```

Comment:
This type is internal and does not need to be included in the API documentation.